src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -103,6 +106,15 @@ public static IDicomServerBuilder AddBackgroundWorkers(this IDicomServerBuilder
                 c.UseApiBehavior = false;
             });
 
+            services.AddVersionedApiExplorer(options =>
+            {
+                options.GroupNameFormat = "'v'VVV";
```

Comment:
lol what am I looking at here? Is this their format string where like `v` and `V` indicate different objects like `y` is year and `H` is hour for `DateTime`?

Reply:
Sorry, its in this format: 'v'major[.minor][-status]
Ill add a comment detailing it.